python/plugins/processing/algs/gdal/GdalUtils.py

```diff
@@ -346,7 +362,7 @@ def ogrConnectionStringAndFormatFromLayer(layer):
                 ogrstr += dsUri.schema() + "."
 
             ogrstr += dsUri.table()
-            format = 'OCI'
+            format = 'OCI'     
```

Comment:
remove trailing space (actually this should be done by running the pre-commit script, as it will apply all formatting options required to get Travis to greenlight the change)